src/Maciek/Fsd.Maciek.Cs/Fsd.Maciek.Cs.Ex4.Web/Models/ProductListWebModel.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Collections.Generic;
+
+namespace Fsd.Maciek.Cs.Ex4.Web.Models
+{
+    public class ProductListWebModel
```

Comment:
Osobiście nigdy nie widziałem XXXWebModel. Nie jest to złe, ale pytanie po co robić tak, jak można XXXModel lub XXXDto